src/prpy/planning/workspace.py

```diff
@@ -206,11 +213,14 @@ def PlanWorkspacePath(self, robot, traj, timelimit=5.0,
                     infeasible_step = True
                     if qnew is not None:
                         # Found an IK
-                        step = abs(qnew - qcurr)
-                        if (max(step) < min_step) and qtraj:
+                        steps = abs(qnew - qcurr) / q_resolutions;
+                        norm = numpy.linalg.norm(steps, ord=norm_order)
+
+                        if (norm < min_step) and qtraj:
```

Comment:
I think this dramatically changed the scaling on `min_step` since the norm is now scaled by DOF resolution. Can you adjust the value of that constant accordingly?